src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1106,17 +1106,15 @@ theorem bsup_eq_blsub {o} (f : Π a < o, ordinal) :
   bsup o f = blsub o f ↔ ∀ a < blsub o f, succ a < blsub o f :=
 by { rw [bsup_eq_sup, blsub_eq_lsub], exact sup_eq_lsub _ }
 
-/-
-lemma blsub_eq_zero {f : Π a < 0, ordinal} : lsub f = 0 :=
-by { rw [←ordinal.le_zero, lsub_le_iff_lt], exact h.elim }
-
-lemma zero_lt_blsub {ι} [h : nonempty ι] (f : ι → ordinal) : 0 < lsub f :=
-h.elim $ λ i, (ordinal.zero_le _).trans_lt (lt_lsub f i)
--/
-
 @[simp] theorem blsub_eq_zero_iff {o} {f : Π a < o, ordinal} : blsub o f = 0 ↔ o = 0 :=
 by { rw [blsub_eq_lsub, lsub_eq_zero_iff], exact out_empty_iff_eq_zero }
 
+lemma blsub_eq_zero {o : ordinal} (ho : o = 0) (f : Π a < o, ordinal) : blsub o f = 0 :=
+by rwa blsub_eq_zero_iff
+
+lemma zero_lt_blsub {o : ordinal} (ho : 0 < o) (f : Π a < o, ordinal) : 0 < blsub o f :=
```

Comment:
```suggestion
lemma blsub_pos {o : ordinal} (ho : 0 < o) (f : Π a < o, ordinal) : 0 < blsub o f :=
```

Reply:
Nice catch! I changed `zero_lt_lsub` to `lsub_pos` as well. Should I restart Bors?